1014-input-design-guidance.md

```diff
@@ -0,0 +1,24 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-15
+updated: 2024-10-21
+---
+
+# Input Design Guidance
+
+[Input](https://clarity.design/documentation/input) component enable the user to input a single sentence or less of text.
+
+## Guidance
+
+- **Must** use Input when you need to allow user to enter a single sentence or less of text.
+- **Must** NOT use Input in case of multiple lines of text. Use [textarea](https://clarity.design/documentation/textarea) component to support multiple lines of text input.
+- **Should** provide a label with a clear and concise text for context. 
```

Comment:
```suggestion
- **Should** provide a clear and concise visible label for context.
- If there is no visible label, an `aria-label` (or other accessible name) **must** be provided and **should** be called out in the design.
```

I think this should be clarified for accessibility.

I made this same comment for the textarea guidance: https://github.com/vmware-clarity/clarity-guidance/pull/34#discussion_r1808969101